core/lib/constants/src/gas.rs

```diff
@@ -0,0 +1,17 @@
+// Currently, every `AGGR_* cost` is overestimated,
+// so there are safety margins around `100_000 -- 200_000`
+
+pub const AGGR_L1_BATCH_COMMIT_BASE_COST: u32 = 242_000;
```

Comment:
Just curious: Would these constants be shared with other crates in the future? If not, could make sense to define them in the modules that use them (this worked previously, right?).